Sources/Support/Windows/Stringify.cpp

```diff
@@ -30,10 +30,10 @@ char const *Stringify::ExceptionCode(DWORD code) {
     DO_STRINGIFY(EXCEPTION_FLT_STACK_CHECK)
     DO_STRINGIFY(EXCEPTION_FLT_UNDERFLOW)
     DO_STRINGIFY(EXCEPTION_ILLEGAL_INSTRUCTION)
-    DO_STRINGIFY(EXCEPTION_IN_PAGE_ERROR)
     DO_STRINGIFY(EXCEPTION_INT_DIVIDE_BY_ZERO)
     DO_STRINGIFY(EXCEPTION_INT_OVERFLOW)
     DO_STRINGIFY(EXCEPTION_INVALID_DISPOSITION)
+    DO_STRINGIFY(EXCEPTION_IN_PAGE_ERROR)
```

Comment:
Wasn't the original sorting the correct sorting? IN comes before INT in a dictionary